app/backend/wells/migrations/0124_create_artesian_conditions_col_update_data_water_639.py

```diff
@@ -0,0 +1,101 @@
+# Generated by Django 2.2.18 on 2021-03-01 19:38
+
+from string import Template
+from django.db import migrations, models
+
+
+WATER_639_USER = 'WATER-639'
+UPDATE_ARTESIAN_CONDITIONS_BASE = Template("""UPDATE $table_name
+    SET artesian_conditions = true,
+        update_user = '$update_user',
+        update_date = now()
+    WHERE artesian_flow > 0 OR artesian_pressure > 0""")
+UPDATE_ACTIVITY_SUBMISSION_ARTESIAN_CONDITIONS = UPDATE_ARTESIAN_CONDITIONS_BASE.substitute(table_name='activity_submission',
+                                                                                            update_user=WATER_639_USER)
+UPDATE_WELL_ARTESIAN_CONDITIONS = UPDATE_ARTESIAN_CONDITIONS_BASE.substitute(table_name='well',
+                                                                             update_user=WATER_639_USER)
+
+UPDATE_ARTESIAN_FLOW_DUMMY_BASE = Template("""UPDATE $table_name
+    SET artesian_flow = NULL,
+        update_user = '$update_user',
+        update_date = now()
+    WHERE artesian_flow = 0.01;
```

Comment:
These dummy values are being cleaned up, but the actual records with a value of `0.01` should also have `artesian_conditions` set to True based on the first SQL statement, correct (looks like it, just double checking)?